snippets/native-platforms/ios-objc/dependencies.md

```diff
@@ -1,4 +1,4 @@
 ```ruby
-pod 'Lock', '~> 1.12'
-pod 'JWTDecode', '~> 0.2'
+pod 'Lock', '~> 1.16'
+pod 'JWTDecode', '0.2'
```

Comment:
3.0 is written in Swift which requires use_frameworks! setting at the Podfile. It breaks Lock-Facebook target compilation unless `Allow Non-modular Includes In Framework Modules` set to YES